internal/environment/get_env_value.go

```diff
@@ -105,3 +107,28 @@ func GetEnvValueIntLog(env string) int {
 
 	return value
 }
+
+// GetAdditionalHttpHeadersFromEnv gets the content of the env var MONACO_ADDITIONAL_HTTP_HEADERS and parses
+// a map of http headers from it, if present. Headers are separated via '\n'
+// Headers consist of a mandatory key part and a value part which can be empty. Key and value part are separated via ':'
+func GetAdditionalHttpHeadersFromEnv() map[string]string {
+	result := make(map[string]string)
+
+	headersString := os.Getenv(AdditionalHttpHeaders)
+	headers := strings.Split(headersString, "\n")
+	for _, header := range headers {
+		headerKeyAndValue := strings.SplitN(header, ":", 2)
+		key := strings.TrimSpace(headerKeyAndValue[0])
+		if key == "" {
+			continue // header key must not be empty
+		}
+
+		if len(headerKeyAndValue) != 2 {
+			result[key] = ""
+		} else {
+			result[key] = strings.TrimSpace(headerKeyAndValue[1])
+		}
```

Comment:
Maybe it is possible to simplify it using `Cut(...)`?
```suggestion
	key, value, _ := strings.Cut(header, ":")
		key = strings.TrimSpace(key)
		if key == "" {
			continue // header key must not be empty
		}

		result[key] = strings.TrimSpace(value)
	}
```